src/actions/AppActions.js

```diff
@@ -215,7 +215,14 @@ export function loginSuccess(auth) {
             }
           });
         })
-        .catch((e) => e)
+        .catch((e) => {
+          if (e.response) {
+            let { status } = e.response;
+            if (status === 401) {
+              window.location.href = '/';
```

Comment:
since we have `dispatch` available can we use routing here ? So use `dispatch(push('/'));`

Reply:
websocket does not reconnect again with correct credentials if you do that. That is the reason I did not used it. Thanks